logging/config/sinks.go

```diff
@@ -187,29 +191,6 @@ func StderrOutput() *OutputConfig {
 	}
 }
 
-func SyslogOutput(tag string) *OutputConfig {
-	return RemoteSyslogOutput(tag, "")
-}
-
-func FileOutput(path string) *OutputConfig {
-	return &OutputConfig{
-		OutputType: File,
-		FileConfig: &FileConfig{
-			Path: path,
-		},
-	}
-}
-
-func RemoteSyslogOutput(tag, remoteUrl string) *OutputConfig {
-	return &OutputConfig{
-		OutputType: Syslog,
-		SyslogConfig: &SyslogConfig{
-			Url: remoteUrl,
-			Tag: tag,
-		},
-	}
-}
```

Comment:
Does this commit remove support for logging to a file or syslog?

Reply:
It is because we dropped our log15 dependency that was providing this. I would like to reintroduce syslog, but I can put file back in their now.